chain-signatures/contract/src/lib.rs

```diff
@@ -646,7 +646,7 @@ impl VersionedMpcContract {
         Self::V0(MpcContract {
             protocol_state: ProtocolContractState::Running(RunningContractState {
                 epoch,
-                participants: Participants { participants },
+                participants: Participants::from_init_participants(participants),
```

Comment:
Would the participant Ids stay the same when we move from one contract to another?

Reply:
Yes, this assign each participant from 0 in order, same as the participant id before the change

Reply:
So this probably won't work for migrations from one contract to another with an already running MPC network that has had multiple reshares. `Participants::from_init_participants` will create a different participant id mapping than the ones the MPC nodes have due to just enumerating over each item and getting their position.

We should have a new parameter for the `account_id_to_participant` value be passable into `init_running` so we can get the same mapping

Reply:
Ah got it! Good point

Reply:
I dropped `Participants::from_init_participants`, now init_running just takes `struct Participants` where node admin can get by view `VersionedMpcContract::state()`